include/time.h

```diff
@@ -54,6 +48,8 @@ extern int clock_getres ( clockid_t clk_id, struct timespec *ts );
 }
 #endif
 
+#endif /* !defined(CLOCK_REALTIME) && !defined(CLOCK_MONOTONIC) */za
```

Comment:
extra characters after the closing "*/"

Reply:
thanks... emacs thick fingers... no fixed.